### ISSUE TITLE
Hardly readable status bar

### ISSUE THREAD

**Issue by 5bentz** [bit-team/backintime#858]
Hello,
Firstly, **thank you bit-team** for this awesome tool. As a _rsync_ lover, I find BiT very convenient to automagically backup my family's computer ! 💯 

As you can see in the picture below, some texts (_Maintenant_, _/home/charles_) are printed in white and highlighted in blue. No problem.
However, the status bar behaves differently. It is also highlighted in blue, but its texts is black, which makes it hardly readable.

![bit_bug_black_on_blue](https://user-images.githubusercontent.com/10729484/34801365-a7b69de8-f668-11e7-81fb-cae686516e89.png)

Regards,

**Comment by ianwillie** [comment]
I had exactly the same problem. I found that by changing the style I could find one that helped.
I am using Xfce and I did this: Xfce>Settings Manager>Appearance>Style . Here I changed Adwaita to Xfce and it seemed to work.

**Comment by 5bentz** [comment]
Interesting, I am using Xfce too.
However, changing the style doesn't do the trick for me. I have tried the five default themes from Xubuntu: Adwaita, High contrast, Greybird, Numix and Raleigh.

I tested from sources the current version from Xubuntu LTS (1.1.24) and the latest version 45ec81c of the next series (1.2 alpha, master branch).

This *bug* does not occur upstream in the 1.2 series.
It seems that d17d176 somewhat *fixed* it...Fishy?
